qt/python/mantidqt/widgets/plotconfigdialog/presenter.py

```diff
@@ -70,6 +70,9 @@ def apply_properties(self):
             if tab and tab.view:
                 tab.apply_properties()
         self.fig.canvas.draw()
+        for tab in reversed(self.tab_widget_presenters):
+            if tab and tab.view:
```

Comment:
The `tab.view` should be removed from this (and the other) loop.
If we have called `close_tab()` on the tab but then not set `tab=None` were already in an inconsistent state where our presenter is living without a view